arci-ros/tests/test_rosrust_utils.rs

```diff
@@ -0,0 +1,79 @@
+#![cfg(target_os = "linux")]
+
+mod util;
+use std::time::SystemTime;
+
+use arci_ros::{convert_ros_time_to_system_time, convert_system_time_to_ros_time};
+
+#[test]
+fn test() {
+    let _ros_core = util::run_roscore_for(util::Language::None, util::Feature::Publisher);
+    // to avoid calling rosrust::init multiple times, call all tests from this function.
+    rosrust::init("test_rosrust_utils");
+    test_convert_ros_time_to_system_time();
+    test_convert_system_time_to_ros_time();
+}
+fn test_convert_ros_time_to_system_time() {
```

Comment:
I don't care actually, but why rustfmt does not warn about this? (a blank line between functions)